pkg/install/install.go

```diff
@@ -20,6 +20,51 @@ import (
 )
 
 var servingVersion = "0.23.0"
+var kourierVersion = "0.23.0"
+
+// Kourier installs Kourier networking layer from Github YAML files
+func Kourier() error {
+
+	fmt.Println("Starting Networking layer install...")
+
+	kourier := exec.Command("kubectl", "apply", "-f", "https://github.com/knative-sandbox/net-kourier/releases/download/v"+kourierVersion+"/kourier.yaml")
```

Comment:
we should put retries on this, I have seeing network issues that first time there are problems getting the yaml from github usually works on the second try after a sleep